src/Sisp.test.js

```diff
@@ -0,0 +1,70 @@
+const Sisp = require('./sisp');
```

Comment:
Why is this file name capital?

Reply:
@nnascim This is a class that was imported

Reply:
It was my mistake 🤦🏽‍♂️ and the code works perfectly with it 🤔

Reply:
File name should be `src/sisp.js`, not `src/Sisp.js`